python_configure.bzl

```diff
@@ -321,6 +324,9 @@ python_configure = repository_rule(
         _PYTHON_BIN_PATH,
         _PYTHON_LIB_PATH,
     ],
+    attrs = {
+        "python3": attr.bool(default=False),
```

Comment:
What about an inverse scenario, where `which python` points to a Python 3 binary and the desire is to use Python 2?

For such flexibility, this attribute could be perhaps be named "python_version" and could take a set of predefined string values (e.g. "2", "3", "default").

Then the logic above would have 4 conditional blocks:

```
if repository_ctx.attr.python_version == "3":
  python_bin_path = repository.ctx.which("python3")
elif repository_ctx.attr.python_version == "2":
  python_bin_path = repository.ctx.which("python2")
elif repository_ctx.attr.python_version == "default":
  python_bin_path = repository.ctx.which("python")
else
  _fail("Invalid python_version value, must be '2', '3', or 'default'.")
```

Sound worthwhile?

Reply:
Makes sense, done.